exercises.md

```diff
@@ -37,6 +37,27 @@ Reflect also on what kind of knowledge you used in this exercise:
 * Knowledge of the domain, of the programming language? Of a framework?
 * What knowledge do you think might be needed to better understand this code?
 
+### Syntax knowledge
+
+The goal of this exercise is to practice to make sure everyone in club is familiar with syntactic elements of the code. 
+
+#### (5 min) Independently - Examine syntax
+
+Look at the code and examine syntactic elements. Do you know the meaning of all elements? You can use these questions as a guide:
+
+* Is it clear to you what the role of each block in the code is (function, condition, repetition etc)?
```

Comment:
is role the right word here?

Reply:
I'll leave it - it mostly works... but there might be a better work, 'function' is not a good one in this case, but probably more what you mean.